### DISCUSSION TITLE
Localised collection's Table view

### DISCUSSION THREAD

**Discussion by HemboPagi** [directus/directus#12178] [Feature Requests]
Hi

Correct me if i am wrong, but at the moment it seems that translated table fields kept in separate table won't be accessible directly from the "layout options". I found that using data template for translation "column" I can make translated filed(s) available, but it is not the same as the context is lost as well user can not use other fields related functionality (sorting for example).

Also when using API i can access all the translated and non-translated data using one API call. Can i go the same with fields metadata?

All the best
Hembo

**Comment by rijkvanzanten** [comment]
I'm not 100% sure what the question is, but try configuring the `translations` display on the `translations` field in your current collection. That will allow you to preview translated values on the parent's table layout

**Reply by HemboPagi** [reply]
Hello

I am thinking of using joined columns as "normal" fields I can select from Layout Options for example

**Reply by azrikahar** [reply]
I'm also not super sure I fully understand the description here, but by `joined columns`, I'm assuming you are referring to the relational fields? If it is, then the good news is the ability to select nested relational fields and show it in the same table was just recently added in #12154! But we will have to wait for the next official release to actually be able to use it.

**Reply by HemboPagi** [reply]
Super! This is exactly what i was after. Sorry for creating confusion.

**Reply by azrikahar** [reply]
No worries! Glad the new feature landed right on time to fit your needs 😄

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
